### ISSUE TITLE
Option to not wrap Action->handle in an transaction

### ISSUE THREAD

**Issue by michielgerritsen** [laravel/nova-issues#180]
When you execute an action, this action is always handled within an transaction. While this is fine for most use cases, there are some edge cases when this is blocking. An easy workaround is to add `DB::commit()` to execute queries anyway, but it would be nicer if we could add something like `->withoutTransaction` on the action.

**Comment by davidhemphill** [comment]
Can you give a specific example to help us get our heads around this?

**Comment by michielgerritsen** [comment]
I had a pretty specific situation: I had to update an model, and then call an api endpoint with that data which checked my application to verify the models data. This was an transaction. The process looked like this:

- Update the model.
- Call the api with data from the model.
- The api calls an api on my end to verify the data.

Because everything is in an transaction i could update the model, but it was not persisted to the database when the api called back.

I fixed this now by calling a simple `\DB::commit();`. Maybe i'm having a too specific use case, i'm not sure ¯\_(ツ)_/¯
